### PR TITLE
Merge 18.8.0.4 beta into `develop`

### PR DESCRIPTION
**Pull request by mokagio**
Got a bunch of conflicts because of deleted files on `develop` that this branch modified, most were localizations because of https://github.com/wordpress-mobile/WordPress-iOS/pull/17630 and https://github.com/wordpress-mobile/WordPress-iOS/pull/17636.

There was also a conflict on the change in the timeout duration of a test, which I solved by keeping the longest timeout value. 

Other than the localizations, the conflicting files were

- `WordPress/WordPressTest/LikeUserHelperTests.swift`
- `WordPress/Classes/ViewRelated/Me/App Settings/About/AboutHeaderView.swift`
- `WordPress/Classes/ViewRelated/Me/App Settings/About/AutomatticAboutScreen.swift`
- `WordPress/Classes/ViewRelated/Me/App Settings/About/AutomatticAppLogosCell.swift`

I ensured the app built and the tests passed before committing 👌

Includes:

- Onboarding flow improvements based on _Call for Testing_ feedback by @momo-ozawa   ([#17643](https://github.com/wordpress-mobile/WordPress-iOS/pull/17643))
- A fix for a crash that became visible in 18.7 by @dvdchr ([#17642](https://github.com/wordpress-mobile/WordPress-iOS/pull/17642))
- A couple of fixes in the About screen by @frosty ([#17646](https://github.com/wordpress-mobile/WordPress-iOS/pull/17646))

### CODE REVIEW

File: WordPress/WordPressTest/LikeUserHelperTests.swift
Comment (mokagio) [inline WordPress/WordPressTest/LikeUserHelperTests.swift]: :thinkingface: This looks unused. I'll get rid of it.

**Review by mokagio — COMMENTED** [review]
- [x] Updates to the localization files (`.strings` and Fastlane metadata) automatically pulled for the new translations that have already been approved in GlotPress
- [x] Version update in `.xcconfig`
- [x] Diffs from the PRs that made it into this beta

**Comment by frosty** [comment]
Thanks @mokagio! Just sharing some context about those About screen files: they have indeed been deleted, as after the code freeze they were replaced with a [Swift package](https://github.com/Automattic/AutomatticAbout-Swift). I'm going to pull the changes from [yesterday's PR](https://github.com/wordpress-mobile/WordPress-iOS/pull/17646#issuecomment-989261071) into the packaged version today, so those changes won't be lost from `develop` before the next freeze.

**Comment by mokagio** [comment]
😭 new conflict...

image

I'm going to solve it here, just in case someone gets to review it soon, even though I'll be running the release finalization in a few hours and possibly superseded this PR. I'd still have to solve the conflict in that case, so the learning is useful regardless.

**Comment by dvdchr** [comment]
@mokagio I can help review this in ~30 minutes. 🙂

**Comment by mokagio** [comment]
@dvdchr thanks mate! That would be helpful 👍 

@momo-ozawa can I also ask you to have a look at how I resolved the conflict in the quick start file here? [`381fc14` (#17652)](https://github.com/wordpress-mobile/WordPress-iOS/pull/17652/commits/381fc14a8be602eb35fbb13f64dcb60d476d1302)

File: WordPress/Classes/ViewRelated/Blog/QuickStartPromptViewController.swift
Comment (dvdchr) [inline WordPress/Classes/ViewRelated/Blog/QuickStartPromptViewController.swift:83]: This is just a very minor, nitpicky note – but the new line is probably not intended here? Considering the `textColor` property belongs to `promptDescriptionLabel`. (Note that this totally is optional and shouldn't block the merge!)

**Review by dvdchr — COMMENTED** [review]
Tested Quick Start according to the steps described in https://github.com/wordpress-mobile/WordPress-iOS/pull/17643. Found some oddities, but I suppose it's more related to https://github.com/wordpress-mobile/WordPress-iOS/pull/17648 rather than this merge conflict? Here's some 👀 on larger font sizes, tested on iPhone 8 Simulator, iOS 14.5, running the `merge/18.8-beta-4-into-develop` branch: 

xxxLarge | AX3 | AX5
--|--|--
17652_xxxLarge | 1652_AX3 | 17652_AX5

- It looks like the updated serif label from #17648 doesn't scale with the font size category.
- On AX5 size, the text seems to overlap with the action buttons, and I couldn't scroll down. Probably because it's not a scroll view? 

Anyways, I'm not sure if this is a known issue or not. Since auto-merge is enabled, I'll defer to @momo-ozawa 🙂 . Hope this helped!

**Comment by mokagio** [comment]
Thanks a lot @dvdchr

File: WordPress/Classes/ViewRelated/Blog/QuickStartPromptViewController.swift
Comment (momo-ozawa) [reply WordPress/Classes/ViewRelated/Blog/QuickStartPromptViewController.swift:83]: > It looks like the updated serif label from #17648 doesn't scale with the font size category.

Ah. You're right :(

@mokagio I'll address the serif scaling in a different PR targeting develop.

> On AX5 size, the text seems to overlap with the action buttons, and I couldn't scroll down. Probably because it's not a scroll view?

I'll embed the text in a scrollview, thanks for pointing that out.

@mokagio Which branch should I target for the scrollview issue? Wondering since the release for 18.8 is very soon.

**Review by momo-ozawa — APPROVED** [review]
The conflict resolution [here](https://github.com/wordpress-mobile/WordPress-iOS/pull/17652#issuecomment-990675857) makes sense to me. I'll address some of the issues @dvdchr found in a separate PR. Thank you @mokagio 🙇‍♀️

**Review by AliSoftware — APPROVED** [review]
I looked specifically at the .strings file and AppLocalizedString.swift in the commit develop->merge-branch where you solved the issues and all seems good 👍 

PS: I thought about also checking that the `pbxproj` merge got resolved properly and especially that the `.strings` files that were removed from disk were also removed from Xcode targets and that `AppLocalizedString.swift` were still properly added to the relevant targets… but I figured if Xcode builds without error on CI that's proof enough that those are correct, otherwise it would have failed on missing file references or unknown symbol.

**Pull request merged**
